src/main/java/org/cactoos/text/UncheckedText.java

```diff
@@ -87,7 +88,7 @@ public UncheckedText(final Text txt, final Func<IOException, String> fbk) {
     public String asString() {
         String txt;
         try {
-            txt = this.text.asString();
+            txt = new IoCheckedScalar<>(this.text::asString).value();
```

Comment:
@pbenety This class can be refactored too: do it yourself or create a new puzzle, please.